R/tax_range_time.R

```diff
@@ -0,0 +1,178 @@
+#' Calculate the temporal range of fossil taxa
+#'
+#' A function to calculate the temporal range of fossil taxa.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences. The
+#' dataframe should contain at least three columns: names of taxa (`character`)
+#' and the maximum (`numeric`) and minimum (`numeric`) age which constrain the
+#' age range (in millions of years) of the fossil occurrence. If the age range
+#' columns are provided as (`character`) values (i.e., stage names), the
+#' function will attempt to extract numeric age data from the
+#' Geological Timescale by linking stage names.
+#' @param name \code{character}. The name of the column you wish to be treated
+#' as the input names (e.g., "species" or "genus").
```

Comment:
```suggestion
#' as the taxonomic names (e.g., "species" or "genus").
```

Reply:
Might be worth leaving this more generic. In reality, the user could supply any names here (e.g. species_1)? What do you think?